aldryn_faq/south_migrations/0022_add_taggit.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
```

Comment:
@Chive waiting for @cyberstar to step in as the django 1.6 migration show no changes using `python src/manage.py schemamigration aldryn_faq -auto`. Probably due to south not recognising the TagManager addition.